### DISCUSSION TITLE
Could the project be trained without GPU, only use CPU? if there is any way, how to config the file?

### DISCUSSION THREAD

**Discussion by haishibei** [STVIR/pysot#350] [General] [answered]
Could the project be trained without GPU, only use CPU? if there is any way, how to config the file?

**Comment by ZhiyuanChen** [comment, marked as answer]
You will have to make tons of edits to make it runs on CPU, and I would highly discourage you to do so as it will take weeks or even months to train.

**Comment by haishibei** [comment]
> You will have to make tons of edits to make it runs on CPU, and I would highly discourage you to do so as it will take weeks or even months to train.

I just want to debug the code not really train because my workstation is not available now. OK, thank you anyway for your reply.
